### DISCUSSION TITLE
The user option needs the class "object-cover"

### DISCUSSION THREAD

**Discussion by KevinEscobarV** [wireui/wireui#335] [BUG]
![image](https://user-images.githubusercontent.com/82073741/174945260-792b75bc-db13-4029-a0b6-dc95edda29b5.png)

**I published the resources to add the class but it doesn't take the changes**

**Comment by ph7jack** [comment]
@KevinEscobarV Could you explain better this discussion? Please, add more details

**Reply by KevinEscobarV** [reply]
In order for the profile photos to look good in the circle they need the object-cover class, as it is seen in the top bar.
![image](https://user-images.githubusercontent.com/82073741/175179876-bca61244-9cf3-4442-bce1-aeea73bbc4b6.png)

I published the resources to add the class in the file "user-option.blade.php" but the changes are not applied.
![image](https://user-images.githubusercontent.com/82073741/175180488-2e55de2f-dd5e-4f04-a958-db33db77455f.png)

_when reviewing the code in the browser, the added class is not found_
